dist/common/scripts/scylla_util.py

```diff
@@ -99,6 +100,27 @@ def scyllabindir():
 def sysconfdir():
     return str(sysconfdir_p())
 
+def get_required_aio_slots():
+    return cpu_count() * 11026 + 65536
+
+def configure_aio_slots():
+    aio_conf_file = "/etc/sysctl.d/aio-max-nr.conf"
+    aio_max_nr_value = "fs.aio-max-nr = " + str(get_required_aio_slots())
+    mode = 'w+'
+    if os.path.exists(aio_conf_file):
+        with open(aio_conf_file, 'r') as file:
+            for line in file:
+                aio_conf_value = re.findall(r"[=]?\d*\.\d+|\d+", line)
+                aio_value = int((', '.join(map(str, aio_conf_value))))
+                if aio_value > get_required_aio_slots():
+                    with open('/proc/sys/fs/aio-max-nr', mode) as f:
+                        f.write(str(aio_value))
+    else:
+        with open(aio_conf_file, mode) as f:
+            f.write(aio_max_nr_value + '\n')
+        with open('/proc/sys/fs/aio-max-nr', mode) as f:
+            f.write(str(get_required_aio_slots()))
+
```

Comment:
Now we write /etc/sysctl.d/aio-max-nr.conf each time we run scylla. That's not good practice, configuration files should be written once and not overwritten.

We also need a %ghost entry in rpm/deb so the file is removed when the system is uninstalled.

I suggest moving back to what we had in 950dbdb4660104a448fab5852ce8ed40e539e1b0, but using a larger value. It's simple.